builder-base/Makefile

```diff
@@ -10,6 +10,8 @@ IMAGE_NAME?=eks-distro/builder
 IMAGE_TAG?=latest
 IMAGE?=$(IMAGE_REPO)/$(IMAGE_NAME):$(IMAGE_TAG)
 
+REPO_ROOT="$(shell cd "$(shell dirname "${BASH_SOURCE[0]}")" && pwd -P)"
```

Comment:
NIT: These don't need to have shell I think, but it should be fine.

Reply:
Also is this actually the repo root, or just the builder-base folder?

Reply:
This would be the builder-base folder, since we're running `make -C builder-base`.